sdk/keyvault/microsoft-azure-keyvault-core/pom.xml

```diff
@@ -51,7 +51,7 @@
   	<dependency>
   		<groupId>com.google.guava</groupId>
   		<artifactId>guava</artifactId>
-    <version>30.1.1-jre</version> <!-- {x-version-update;com.google.guava:guava;external_dependency} -->
+    <version>30.0-jre</version> <!-- {x-version-update;com.google.guava:guava;external_dependency} -->
```

Comment:
We don't need to downgrade these dependencies for Key Vault as the CVE in question was fixed in version 30.0 and onwards.